content/developers/_index.md

```diff
@@ -8,4 +8,52 @@ images: []
 aliases:
     - /docs/developers/
 ---
-Information for developers can be found here:
\ No newline at end of file
+<div class= "row justify-content-center">
+    <div class="col-md-12 col-lg-10 col-xl-10">
+    <h1>Developers</h1>
+    <p>If you are a developer who is looking to easily add provenance to their data, this section is for you. <br>
+    You may be building a new application or looking for a way to add functionality to something that you already use every day.</p>
+    <p>The RKVST API or the YAML runner provide a simple way for you to integrate a provenance layer into your existing data platform so that you do not need to change the way that your users work.</p>
```

Comment:
```suggestion
    <p>The RKVST REST API, python SDK, or the YAML runner provide a simple way for you to integrate a provenance layer into your existing data platform so that you do not need to change the way that your users work.</p>
```